api/games/create.js

```diff
@@ -184,56 +184,141 @@ export const createGameFromLobby = gameLobby => {
   let stageIndex = 0;
   let totalDuration = 0;
   let firstRoundId;
-  params.roundIds = params.rounds.map((round, index) => {
-    const roundId = Rounds.insert(_.extend({ gameId, index }, round), {
-      autoConvert: false,
-      filter: false,
-      validate: false,
-      trimStrings: false,
-      removeEmptyStrings: false
-    });
-    const stageIds = round.stages.map(stage => {
+
+  const insertOption = {
+    autoConvert: false,
+    filter: false,
+    validate: false,
+    trimStrings: false,
+    removeEmptyStrings: false
+  };
+
+  let RoundsOp = Rounds.rawCollection().initializeUnorderedBulkOp();
+  let StagesOp = Stages.rawCollection().initializeUnorderedBulkOp();
+  let roundsOpResult;
+  let stagesOpResult;
+
+  params.rounds.forEach((round, index) =>
+    RoundsOp.insert(
+      _.extend(
+        {
+          gameId,
+          index,
+          _id: Random.id(),
+          createdAt: new Date(),
+          updatedAt: new Date()
+        },
+        round
+      ),
+      insertOption
```

Comment:
I don't think these insertOption apply to Bulk ops, those options have to do with SimpleSchema, which is not part of Mongo.